dll/cpl/desk/background.c

```diff
@@ -1140,6 +1142,43 @@ SetDesktopBackColor(HWND hwndDlg, PBACKGROUND_DATA pData)
     RegCloseKey(hKey);
 }
 
+static VOID
+OnCustomButton(HWND hwndDlg, PBACKGROUND_DATA pData)
+{
+    HPROPSHEETPAGE hpsp[1] = {0};
+    PROPSHEETHEADER psh;
+    PROPSHEETPAGE psp;
```

Comment:
If you do `PROPSHEETHEADER psh = {sizeof(psh)}` then you can skip both the ZeroMemory and the `psh.dwSize = ...`